notebooks/02-quality-flags/TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,1011 @@
+{
```

Comment:
<p>We could motivate this by pointing out the "dips" in the light curve above, and asking, are they real signals, at about 2.5 day intervals?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>okay will do</p>

Reply:
<p>I agree we should motivate it this way. We can also add that with ~12 days of data, the largest period we can detect is 6 days (that might also clean up our plot)</p>

Reply:
<p>added text to motivate this code block, both of these points included now</p>